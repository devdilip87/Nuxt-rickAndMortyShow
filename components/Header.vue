<template>
    <div class="icon-bar">
    <span class="active"><i class="fa fa-home"></i></span>
    <h3>Rick And Morty Show</h3>
    </div>
</template>

<style>
    .icon-bar {
        width: 100%;
        background-color: #555;
        overflow: auto;
    }

    .icon-bar span {
        float: left;
        width: 50px;
        text-align: center;
        padding: 10px 0;
        color: white;
        font-size: 20px;
    }

    .active {
        background-color: #2196F3;
    }

    .icon-bar h3 {
        color: white;
        text-align: center;
        margin-top: 12px;
    }
</style>