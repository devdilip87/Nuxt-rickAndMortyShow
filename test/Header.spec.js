import { mount, shallowMount } from '@vue/test-utils';
import Header from '@/components/Header.vue';

describe('Header', () => {
    test('is a Vue instance', () => {
        const wrapper = shallowMount(Header)
        expect(wrapper.isVueInstance()).toBeTruthy()
    })

    test('it generate snapshot test', () => {
        const wrapper = shallowMount(Header)
        expect(wrapper.html()).toMatchSnapshot()
    })
})