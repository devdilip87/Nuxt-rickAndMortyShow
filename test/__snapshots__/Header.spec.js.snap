// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header it generate snapshot test 1`] = `
"<div class=\\"icon-bar\\"><span class=\\"active\\"><i class=\\"fa fa-home\\"></i></span>
  <h3>Rick And Morty Show</h3>
</div>"
`;
